mail_notification_custom_subject/tests/test_mail_notification_custom_subject.py

```diff
@@ -140,6 +140,7 @@ def test_email_subject_template_w_original(self):
         self.assertEquals(mail_message_1.subject, "Test and something more")
 
     def test_bad_template_does_not_break(self):
+        # Create template with error (obaject) to test error
```

Comment:
This is not a docstring, but a comment. A single line docstring is:
```suggestion
        """Create template with error (obaject) to test error."""
```